Jenkinsfile_nightly

```diff
@@ -24,7 +24,7 @@ withNightlyPipeline(type, product, component) {
     enableSlackNotifications(channel)
 
     enableFullFunctionalTest(30)
-    enableSecurityScan()
```

Comment:
@paul-graham - looks like this is bringing back a commented-out line?

Reply:
The commit shows that @sreekanthpuligadda simply commented it out, rather than introducing a commented-out line. The commit message for this reads:

> disabled security scan because of false positives causing nightly build failures. api test user details update to validate oidc or oauth as per configuration

It seems possible that it'll come back in again at some point, which would explain why it's _disabled_ rather than removed.

Reply:
Ah, OK - thanks for the explanation, @paul-graham.